catalyst/dl/callbacks/criterion.py

```diff
@@ -57,7 +87,79 @@ def on_batch_end(self, state: RunnerState):
             }
         )
 
-        self._add_loss_to_state(state, loss)
+        _add_loss_to_state(self.prefix, state, loss)
+
+
+class CriterionAggregatorCallback(Callback):
+    """
+    This callback allows you to aggregate the values of the loss
+    (by ``sum`` or ``mean``) and put the value back into ``state.loss``.
+    """
+    def __init__(
+        self,
+        prefix: str,
+        loss_keys: List[str] = None,
+        loss_aggregate_fn: str = "sum",
```

Comment:
by default, it's better to use mean aggregation to prevent batch size dependency

Reply:
disagree, the sum of the losses is more frequent (e.g. adding different regularizations)

Reply:
if the size is a reason, you could add a multiplier to each loss and the aggregated loss